packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -180,6 +180,33 @@ export class ConfigUtil {
     return info.aliases.getUsername(usernameOrAlias) || usernameOrAlias;
   }
 
+  public static async unsetTargetOrg(usernameOrAlias: string) {
+    const originalDirectory = process.cwd();
+    // In order to correctly setup Config, the process directory needs to be set to the current workspace directory
+    const workspacePath = workspaceUtils.getRootWorkspacePath();
+    try {
+      // checks if the usernameOrAlias is non-empty and active.
+      if (usernameOrAlias) {
```

Comment:
@CristiCanizales Thank you for creating a new function. I was wondering, given target  org is a config setting, why this check is necessary, or even needed as a parameter?. Throwing when the org is expired could be too much, given that unsetting target org is what is necessary. What do you think?

Reply:
Yeah you're right. I was thinking you needed to have the right connection to the default org in order to logout, but that's handled apart so we don't need it in this new unset function. thanks!